src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1889,10 +1889,10 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
                     visibilityTable, responseStatus, roster, responseComments);
         }
         
-        if (params.get("questionNum") != null) {
-            int questionNumber = Integer.parseInt(params.get("questionNum"));
+        if (params.get("questionId") != null) {
+            String questionId = params.get("questionId");
             FeedbackQuestionAttributes question = fqLogic.getFeedbackQuestion(
-                    feedbackSessionName, courseId, questionNumber);
+                                                                    questionId);
```

Comment:
It does not make sense to get the feedback question if questionId is QUESTION_NUM_FOR_RESPONSE_RATE, then check for null later. Can create a separate issue to fix this logic.